code/datums/actions/mobs/projectileattack.dm

```diff
@@ -250,3 +279,51 @@
 	owner.visible_message(span_danger("[owner] fires the proto-kinetic accelerator!"))
 	owner.face_atom(target_atom)
 	new /obj/effect/temp_visual/dir_setting/firing_effect(owner.loc, owner.dir)
+
+/datum/action/cooldown/mob_cooldown/projectile_attack/colossus_final
+	name = "Titan's Finale"
+	desc = "A single-use ability that shoots a large amount of projectiles around you."
+	cooldown_time = 2.5 SECONDS
+
+/datum/action/cooldown/mob_cooldown/projectile_attack/colossus_final/Activate(atom/target_atom)
+	StartCooldown(30 SECONDS)
+	attack_sequence(owner, target_atom)
+	StartCooldown()
+	Remove(owner)
+
+/datum/action/cooldown/mob_cooldown/projectile_attack/colossus_final/attack_sequence(mob/living/firer, atom/target)
+	var/mob/living/simple_animal/hostile/megafauna/colossus/colossus
+	if(istype(firer, /mob/living/simple_animal/hostile/megafauna/colossus))
+		colossus = firer
+
+	var/finale_counter = 10
+	for(var/i in 1 to 20)
+		if(finale_counter > 4 && colossus)
+			colossus.say("Die!!", spans = list("colossus", "yell"))
```

Comment:
Maybe go for a "Perish." or a "FALL." instead of die, it's not nearly as grand for a final attack, since all of his other attacks have him parroting the same lines.

Reply:
true the die with exclamations kinda makes him sound like a mad baby